catboost/python-package/catboost/core.py

```diff
@@ -5826,3 +5826,71 @@ def to_classifier(model):
     for attr in model.__dict__:
         setattr(classifier, attr, getattr(model, attr))
     return classifier
+
+
+from catboost.utils import read_cd
+
+
+class ColumnDescription:
```

Comment:
It will be nice to have interface like this:
```
columns = ColumnDescription(columns_count=10)
columns[0].type = EColumnType.Label
columns[1].type = EColumnType.GroupId
columns[5].type = EColumnType.Categ
columns[5].name = "Feature name"
```